giving-example/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@{
+    ViewData["Title"] = "Adyen Checkout Home";
+}
+<div class="main-container">
+  <div class="info">
+    <h1>Adyen Giving - Donations demo</h1>
+    <div class="giving-view-container">
+    <p>
+        This is a sample designed to demonstrate the <a href="https://docs.adyen.com/online-payments/donations/web-component">Adyen Giving</a> donations workflow.
+    </p>
+    <p>
+        First make a test payment using one of our <a href="https://docs.adyen.com/development-resources/test-cards/test-card-numbers">test card numbers</a>, so you can see the donation screen appear.
+    </p>
+    <p>
+        To learn more about Adyen's giving feature, check out <a href="https://docs.adyen.com/online-payments/donations/">donations.</a>
```

Comment:
I have removed this in Node. Isn't one link to `https://docs.adyen.com/online-payments/donations` enough?

Reply:
I've removed this line